src/loader.js

```diff
@@ -44,10 +44,11 @@ export class Loader {
     data: Array<number>,
   ): Promise<PublicKey> {
     {
+      const balanceNeeded = await connection.getMinimumBalanceForRentExemption(data.length);
       const transaction = SystemProgram.createAccount(
         payer.publicKey,
         program.publicKey,
-        1,
+        balanceNeeded > 0 ? balanceNeeded : 1,
```

Comment:
when do you see `balanceNeeded < 1`?

Reply:
On a cluster not charging rent, as those in our current solana docker images.
Perhaps that will never exist on any cluster going forward, but not knowing whether that might be configurable per cluster, I thought this seemed less brittle.